distributed/hdfs.py

```diff
@@ -21,7 +22,7 @@
 
 
 def walk_glob(hdfs, path):
-    if '*' not in path and hdfs.info(path)['kind'] == 'directory':
+    if '*' not in path and hdfs.info(path).get('kind', '') == 'directory':
```

Comment:
My apologies, this should be reverted - it is a remnant of trying to get the hdfs read_bytes to work with s3fs directly